reflex/components/graphing/recharts/charts.py

```diff
@@ -0,0 +1,457 @@
+    cy: Var[Union[int, str]]
+
+    # The angle of first radial direction line.
+    start_angle: Var[int]
+
+    # The angle of last point in the circle which should be startAngle - 360 or startAngle + 360. We'll calculate the direction of chart by 'startAngle' and 'endAngle'.
+    end_angle: Var[int]
+
+    # The inner radius of first circle grid. If set a percentage, the final value is obtained by multiplying the percentage of maxRadius which is calculated by the width, height, cx, cy. Number | Percentage
+    inner_radius: Var[Union[int, str]]
+
+    # The outer radius of last circle grid. If set a percentage, the final value is obtained by multiplying the percentage of maxRadius which is calculated by the width, height, cx, cy. Number | Percentage
+    outer_radius: Var[Union[int, str]]
+
+    # The gap between two bar categories, which can be a percent value or a fixed value. Percentage | Number
+    bar_category_gap: Var[str]
+
+    # The gap between two bars in the same category, which can be a percent value or a fixed value. Percentage | Number
+    bar_gap: Var[str]
+
+    # The size of each bar. If the barSize is not specified, the size of bar will be calculated by the barCategoryGap, barGap and the quantity of bar groups.
+    bar_size: Var[int]
+
+    # Valid children components
+    valid_children: List[str] = [
+        "PolarAngleAxis",
+        "PolarRadiusAxis",
+        "PolarGrid",
+        "Legend",
+        "GraphingTooltip",
+        "RadialBar",
+    ]
+
+    def get_event_triggers(self) -> dict[str, Union[Var, Any]]:
+        """Get the event triggers that pass the component's value to the handler.
+
+        Returns:
+            A dict mapping the event trigger to the var that is passed to the handler.
+        """
+        return {
+            EventTriggers.ON_CLICK: lambda: [],
+            EventTriggers.ON_MOUSE_ENTER: lambda: [],
+            EventTriggers.ON_MOUSE_MOVE: lambda: [],
+            EventTriggers.ON_MOUSE_LEAVE: lambda: [],
+        }
+
+
+class ScatterChart(ChartBase):
+    """A Scatter chart component in Recharts."""
+
+    tag = "ScatterChart"
+
+    # Valid children components
+    valid_children: List[str] = [
+        "XAxis",
+        "YAxis",
+        "ZAxis",
+        "ReferenceArea",
+        "ReferenceDot",
+        "ReferenceLine",
+        "Brush",
+        "CartesianGrid",
+        "Legend",
+        "GraphingTooltip",
+        "Scatter",
+    ]
+
+    def get_event_triggers(self) -> dict[str, Union[Var, Any]]:
+        """Get the event triggers that pass the component's value to the handler.
+
+        Returns:
+            A dict mapping the event trigger to the var that is passed to the handler.
+        """
+        return {
+            EventTriggers.ON_CLICK: lambda: [],
+            EventTriggers.ON_MOUSE_UP: lambda: [],
+            EventTriggers.ON_MOUSE_DOWN: lambda: [],
```

Comment:
not sure how these are different than `on_click` if the event handler doesn't take any args, like _where_ or _what_ the user mouse_down'd on

Reply:
I agree these function the same I initially added because they were used in recharges, but removing would be the best choice for simplicity.